### ISSUE TITLE
RPC failure when pushing local repo

### ISSUE THREAD

**Issue by cgsecret** [go-gitea/gitea#13338]
- Gitea version (or commit ref):1.13.0
- Git version:2.29.0
- Operating system: Ubuntu Client, Host Docker
  
  
  
- Database (use `[x]`):
  - [ ] PostgreSQL
  - [ ] MySQL
  - [ ] MSSQL
  - [ ] SQLite
- Can you reproduce the bug at https://try.gitea.io:
  - [ ] Yes (provide example URL)
  - [ ] No
- Log gist:

## Description

...

Hi

i am trying to add a git repo to gitea. The repo was a SVN repo i converted with git svn. 
I am using the http:example.com/reponame scheme not ssh.
  ```
git remote add origin <targeturl>+reponame
git push --set-upstream origin master
<authentication prompt>
Enumerating objects: 87, done.
Counting objects: 100% (87/87), done.
Delta compression using up to 8 threads
Compressing objects: 100% (75/75), done.
Writing objects: 100% (87/87), 3.37 MiB | 127.80 MiB/s, done.
Total 87 (delta 21), reused 0 (delta 0), pack-reused 0
error: RPC failed; HTTP 404 curl 22 The requested URL returned error: 404
fatal: the remote end hung up unexpectedly
fatal: the remote end hung up unexpectedly
Everything up-to-date



```
I added all permission needed to create repos and push local repos to gitea and with original git repos everything is working fine. Im not understanding the RPC failure. 

Thanks for ur help and please ask for further informations on this issue.

**Comment by zeripath** [comment]
The likely issue here is an incorrect `LOCAL_ROOT_URL` in the `[server]` section of your app.ini - however you have not, despite the issue proforma's request, given us any gitea logs to help clarify that situation.

It really is important to provide pertinent logs

Please read https://docs.gitea.io/en-us/logging-configuration/#debugging-problems

In addition, if your problem relates to git commands set `RUN_MODE=dev` at the top of app.ini

**Comment by cgsecret** [comment]
Hi
i was hesitant providing the logs since it does contain sensible data like the repo name and the username. Here the part i hope is related to the error with (i hope all) the sensible data removed.
I specified LOCAL_ROOT_URL which was not in my app.ini. I only had ROOT_URL.  I also added RUN_MODE=dev and LEVEL=debug for the log.
I tried the domain name and the local IP adress for local_root_url but the behavior does not change.
[GiteaLog.txt](https://github.com/go-gitea/gitea/files/5459346/GiteaLog.txt)

Thank you very much!
